src/test/coins_tests.cpp

```diff
@@ -273,8 +341,6 @@ BOOST_AUTO_TEST_CASE(anchors_test)
     }
 }
 
-BOOST_FIXTURE_TEST_SUITE(coins_tests, BasicTestingSetup)
```

Comment:
It's just being moved around, because the other tests were not placed under the test fixture. (It didn't matter in practice but when I was manually running `coins_tests` it wasn't running them.)